src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -603,7 +603,7 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
     }));
 
     return (
-      <EuiPageContent horizontalPosition="center">
+      <div>
```

Comment:
maybe `<React.Fragment>` is better than using `<div>` here?
same applies for the [saved_object_view page](https://github.com/elastic/kibana/pull/102281/files#diff-20257a35467392454961fa1044dfcc249dd944d776ff709c73fd3435c8dce2b4R89)